tests/api_test.py

```diff
@@ -947,6 +947,12 @@ def test_jvp_mismatched_arguments(self):
       TypeError,
       "primal and tangent arguments to jax.jvp do not match.",
       lambda: api.jvp(lambda x: -x, (np.float16(2),), (np.float32(4),)))
+    # If primals and tangents are not of the same shape then raise error
+    fun = lambda x: x+1
+    with self.assertRaisesRegex(ValueError, "jvp called with inconsistent primal and tangent shapes"):
+      api.jvp(fun, (jnp.array([1.,2.,3.]),), (jnp.array([1.,2.,3.,4.]),))
```

Comment:
When want to test exceptions, each statement that you want to test must be in their own `self.assertRaisesRegex`, otherwise you are only testing the first one.

Reply:
I see, I will make the change. Thanks